### DISCUSSION TITLE
adding timestamp

### DISCUSSION THREAD

**Discussion by mues-lee** [theengs/gateway#100] [Ideas]
Hello,

I believe it would be great to have the option to add a timestamp into the outgoing MQTT messages.

couldn't find if this was already requested, so please feel free to delete this idea in that case.

Best regards
mues-lee

**Comment by koenvervloesem** [comment]
This question has come up in the past, but if I remember correctly we decided to not include timestamps because it would make the payloads bigger and timestamps are easy to generate (or even already available) at the receiving end.

Is there a specific environment where you need these timestamps and where it's difficult to get these?

**Reply by mues-lee** [reply]
I understand, wasn't aware that the size of the payload might be a problem.
Then i will continue to create the timestamp with openHAB, was just my OCD that prefers to have all the data from one source :)
